### DISCUSSION TITLE
Customizing CP Navigation

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5688] [Q&A]
I just customized the navbar and add new menu in AppServiceProvider in boot function like that : 
Nav::extend(function ($nav) {
            $nav->content('Notification')
                    ->url('notification')
                    ->icon('shopping-cart');
        });
And created new controller and set the route successfully but it does not appear in system permissions so i can grant it to specific users. How to handle it to be shown in permissions please

**Comment by duncanmcclean** [comment]
It won't get permissions automatically. That's something you have to do yourself - [review the docs](https://statamic.dev/extending/permissions#basic-permissions).

**Reply by HazemMohamed252** [reply]
It is first time to treat with Policy and i can not understand it, can you please show me any guidline to help me ? Thanks

**Reply by jasonvarga** [reply]
Read the entire page, there are links to the appropriate Laravel docs too which will give more information.

**Reply by HazemMohamed252** [reply]
I tried but nothing happen. all i want is to control the new nav item that i added in AppServiceProvider : 
Nav::extend(function ($nav) {
            $nav->content('Notification')
                    ->url('notification')
                    ->icon('shopping-cart');
        });
        
Any clear way please because it is first time to treat with Statamic
